opensrp-pnc/src/main/java/org/smartregister/pnc/PncLibrary.java

```diff
@@ -238,4 +260,11 @@ public PncRulesEngineHelper getPncRulesEngineHelper() {
     protected Date getDateNow() {
         return new Date();
     }
+
+    public AppExecutors getAppExecutors() {
```

Comment:
If this is here, kindly replace all the other uses of `AppExecutors` in the app to use it, instead of local instantiation in classes.

Reply:
refactor all AppExecutors with this

Reply:
yeah `PncLibrary.getInstance().getAppExecutors()`